pnpm-workspace.yaml

```diff
@@ -1,3 +1,4 @@
 packages:
   - "apps/*"
-  - "packages/*"
\ No newline at end of file
+  - "packages/*"
+  - "packages/config/*"
```

Comment:
That surprises me. Is that necessary to reference the eslint in the root?

Reply:
I moved all the config packages into one level deeper (not very noticeable on GitHub diff), so now it's `packages/config/eslint`, packages/config/tailwind etc. I think it makes it cleaner when you browse the file structure.

So we need to tell pnpm to look for packages in that directory too, an alternate way would be to make the above one `packages/**` so it would take all levels. Personal preference there too whichever one you prefer

Reply:
Ah, got it. Works for me. 👍